src/pages/journal/journal.effects.ts

```diff
@@ -91,13 +91,7 @@ export class JournalEffects {
             ),
             catchError((err: HttpErrorResponse) => {
               // For HTTP 304 NOT_MODIFIED we just use the slots we already have cached
-              if (err.status !== 304) {
-                this.store$.dispatch(new SaveLog(
-                  this.logHelper.createLog(LogType.ERROR, 'Retrieving Journal', err.message),
-                ));
-              }
-
-              if (err.status === 304 || err.message === 'Timeout has occurred') {
+              if (err.status === 304) {
```

Comment:
There's an enum with the HTTP status codes in that you could use that would keep the code consistent

`/src/shared/models/http-status-codes.ts`
```
export enum HttpStatusCodes {
  OK = 200,
  CREATED = 201,
  ACCEPTED = 202,
  PARTIAL_CONTENT = 206,
  NOT_MODIFIED = 304,
  BAD_REQUEST = 400,
  UNAUTHORIZED = 401,
  FORBIDDEN = 403,
  NOT_FOUND = 404,
  CONFLICT = 409,
  UNPROCESSABLE_ENTITY = 422,
  INTERNAL_SERVER_ERROR = 500,
}
```